packages/lib/src/common/utils/index.js

```diff
@@ -6,6 +6,7 @@ import {
   isPreviewMode,
   isFormFactorMobile,
 } from '../window/viewModeWrapper';
+import GALLERY_CONSTS from '../constants'
```

Comment:
I prefer to avoid backward imports (`import x from '../../'`) since it makes the file less portable

Reply:
is there a way to avoid it?